stix_shifter/stix_translation/src/modules/bigfix/bigfix_to_stix/bigfix_to_stix_translator.py

```diff
@@ -0,0 +1,214 @@
+import re
```

Comment:
We probably want to avoid copying the entire json_to_stix_translator file.

Reply:
there are lot of changes i had to make inside this file. i also had to reformat the json. didn't want to break the base translator at the first place. you can have a look at this translator for now. i'm working on to make it compatible with base  json_to_stix_translator if i can avoid copying the code i will push new changes

Reply:
Did you look at what I did for the QRadar module? I defined a class and method in the qradar_utils.py file, passed that into the JSONToStix call in qradar_translator (this would be bixfix_translator in your case) and then use that passed-in class to handle any QRadar specific logic.

Reply:
so i haven't created any util class but formatted the output in transmission `bigfix_results_connector.py` so the translation interface will get well formatted json so that we can use our base json_to_stix classes.